### DISCUSSION TITLE
Identity linked messages

### DISCUSSION THREAD

**Discussion by simontakouridis** [idena-network/idena-docs#147] [IIPs]
I would like to link my Idena identity to my Twitter social media profile to give users better confidence that I am not a bot. I have included the url to my Idena identity page at scan.idena.io in my Twitter profile, however there is no evidence that the Idena identity is actually mine. As such, I would like to have a method to add my Twitter username to my Idena identity page. One such way would be to have a mechanism of "Identity linked messages" that are signed through the Idena client, stored on the blockchain, and automatically published on my Idena identity page - perhaps the 10 most recent "Identity linked messages" can be published to prevent spam.

With this change, users could navigate from my Twitter profile to my Idena identity page and verify that my twitter username is correct, and therefore enhancing my proof of personhood in the social-sphere.

**Comment by Zen-44** [comment]
You could have a smart contract where you store your twitter tag (and since your identity sends the tx, it has the necessary signature). 
Without some intervention from the twitter side any identity could claim any twitter account. This approach would only work if you put your identity on the twitter page too like you said.

**Reply by simontakouridis** [reply]
That's a good idea. Thanks.

**Comment by slavko989** [comment]
Nice idea, something could be done about that. A way to verify human status and show that on other platforms as a proof is a game changer definitely.
